Swift/AEPSampleApp/xdmlib/MobileSDKCommerceSchema.swift

```diff
@@ -36,15 +35,13 @@ public struct MobileSDKCommerceSchema
 	public var commerce: Commerce?
 	public var eventMergeId: String?
 	public var eventType: String?
-	public var identityMap: IdentityMap?
```

Comment:
Open to discussion on this one. We may still need examples with custom identityMap per event, but the common use-case would use the same identityMap for all experience events.

Reply:
It is a valid element in the schema so it is not wrong to include it. We are not using it in this app, so we can remove it for the sake of cutting dead code. However in another app there can be valid cases where it would be used, such as when multiple datasets are used and each may need different identifiers.